### DISCUSSION TITLE
Welcome to discord-group-spammer Discussions!

### DISCUSSION THREAD

**Discussion by FuckingToasters** [FuckingToasters/discord-group-spammer#6] [Announcements]
## 👋 Welcome to the Group-Spammer Discussions!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yung-rajan** [comment]
why i have this
![image](https://user-images.githubusercontent.com/106391041/170717380-c102b6a6-b55f-4023-8cf3-91fd6abe45db.png)

**Reply by FuckingToasters** [reply]
I'm not sure why you have this. Maybe you can open a issue and include more things like code and traceback (if any)
